modules/nf-core/hhsuite/hhblits/environment.yml

```
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - "bioconda::hhsuite=3.3.0"
```

Not a big thing, but afaik usually we don't have quotes around packages

```suggestion
  - bioconda::hhsuite=3.3.0
```